src/test/java/com/google/cloud/spanner/jdbc/JdbcParameterStoreTest.java

```diff
@@ -438,8 +463,15 @@ public void testSetInvalidParameterWithType() throws SQLException, IOException {
       assertInvalidParameter(params, Boolean.TRUE, type);
     }
 
+    // types that should lead to float32
```

Comment:
Should the comment be `types that should *not* lead to float32`?

Reply:
Yes, copy paste error